test/drop-down-menu-spec.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react/addons';
+import DropDownMenu from 'drop-down-menu';
+import injectTheme from './fixtures/inject-theme';
+
+const TestUtils = React.addons.TestUtils
+
+describe('DropDownMenu', () => {
+    let ThemedDropdownMenu;
+
+    beforeEach(() => {
+        ThemedDropdownMenu = injectTheme(DropDownMenu);
+    });
+
+    it(`Issue #1761`, () => {
```

Comment:
Thanks @yulric! This looks good, except could you change the name of this unit test? So if `x` is the index of the selected value, then maybe something like:

`shows value at index x as selected by default`

I'll go ahead and merge this then :)